jeremy2918/tsseract
# Changed img to next Image

# [Use NextJS Image](https://trello.com/c/bu4gc1VI/77-i-want-to-use-the-nextjs-image-for-all-the-images-in-the-app-so-that-the-optimization-is-at-the-top)


This PR tries to make better use of images by implementing NextJS Images to the app. It is only using it in the places where the image is in the `static` folder of the app

## Testing Steps

- Pull and checkout to this branch.
- `npm install`
- `npm run tsc`
- `npm test` All tests should be passing successfully.
- `npm run dev`
- Check that the home image is already loaded on the screen when it charges.
